src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public int getRandomBall() {
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. This method should return instance of class Ball.

Reply:
Each call `getRandomBall()` will create `new Random()` object. Try to avoid this by declaration of this variable at the class-level.